hunts/src/HuntViewMain.js

```diff
@@ -1,13 +1,21 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { useSelector, useDispatch } from "react-redux";
-import { fetchPuzzles, selectPuzzleTableData } from "./puzzlesSlice";
+import {
+  fetchPuzzles,
+  selectPuzzleTableData,
+  getNumUnlocked,
+  getNumSolved,
+  getNumUnsolved,
+  getNumMetasSolved,
```

Comment:
nit: our other selectors are prefixed `select` instead of `get` so I might rename them that